doc/release-notes/6.5-release-notes.md

```diff
@@ -0,0 +1,385 @@
+
+### Harvesting "oai_dc" Metadata Prefix When Extended With Specific Namespaces
+
+Some data repositories extend the "oai_dc" metadata prefix with specific namespaces. In this case, harvesting of these datasets into Dataverse was not possible because an XML parsing error was raised.
+
+Harvesting of these datasets has been fixed by excluding tags with namespaces that are not "dc:". That is, only harvesting metadata with the "dc" namespace. See #10837.
+
+### Harvested Dataset PID from Record Header
+
+When harvesting, Dataverse can now use the identifier from the OAI-PMH record header as the persistent id for the harvested dataset.
+
+This will allow harvesting from sources that do not include a persistent id in their oai_dc metadata records, but use valid DOIs or handles as the OAI-PMH record header identifiers.
+
+It is also possible to optionally configure a harvesting client to use this OAI-PMH identifier as the **preferred** choice for the persistent id. See the [Harvesting Clients API](https://guides.dataverse.org/en/6.5/api/native-api.html#create-a-harvesting-client) section of the Guides, #11049 and #10982 for more information.
+
+### Harvested Datasets Can Have Multiple "otherId" Values
+
+When harvesting using the DDI format, datasets can now have multiple "otherId" values. See #10772.
+
+### Multiple Languages in Docker
+
+Documentation has been added to explain how to set up multiple languages (e.g. English and French) in the tutorial for setting up Dataverse in Docker.
+
+See [the tutorial](https://guides.dataverse.org/en/6.5/container/running/demo.html#multiple-languages), #10939, and #10940.
+
+### GlobusBatchLookupSize
+
+An optimization has been added for the Globus upload workflow, with a corresponding new database setting: `:GlobusBatchLookupSize`
+
+See the [Database Settings](https://guides.dataverse.org/en/6.5/installation/config.html#GlobusBatchLookupSize) section of the guides, #10977, and #11040 for more information.
+
+## Bugs Fixed
+
+### Relation Type (Related Publication) and DataCite
+
+The subfield "Relation Type" was added to the field "Related Publication" in Dataverse 6.4 (#10632) but couldn't be used without workarounds described in an [announcement](https://groups.google.com/g/dataverse-community/c/zlRGJtu3x4g/m/GtVZ26uaBQAJ) about the problem. The bug has been fixed and workarounds and are longer required. See #10926 and the announcement above.
+
+### Sort Order for Files
+
+"Newest" and "Oldest" were reversed when sorting files on the dataset landing page. This has been fixed. See #10742 and #11000.
+
+### Guestbook Email Validation
+
+In the Guestbook UI form, the email address is now checked for validity. See #10661 and #11022.
+
+### Updating Files Now Possible When Latest and Only Dataset Version is Deaccessioned
+
+When a dataset was deaccessioned and was the only previous version it would cause an error when trying to update the files. This has been fixed. See #9351 and #10901.
```

Comment:
```suggestion
When a dataset was deaccessioned, and was the only previous version, it would cause an error when trying to update the files. This has been fixed. See #9351 and #10901.
```